src/index.js

```diff
@@ -88,18 +92,41 @@ function visitEntity(entity, entitySchema, bag, options) {
   let normalized = visitObject(entity, entitySchema, bag, options);
   mergeIntoEntity(stored, normalized, entityKey);
 
+  if(entitySchema._mappedBy) {
+    const fk = entitySchema._mappedBy;
+    if(entitySchema[fk] instanceof IterableSchema && parentId) {
+      if(!stored[fk]) {
+        stored[fk] = [];
+      }
+      stored[fk] = Array.from(new Set([
```

Comment:
In any case, it must be too much for what I want to achieve. This could work too : 
```js
stored[fk] = stored[fk] || [];
if (stored[fk].indexOf(parentId) === -1) {
  stored[fk] = [...stored[fk], parentId];
}
```